openpaas-james/mailbox/opendistro-tests/src/test/resources/eml/mailWithHeaders.eml

```diff
@@ -0,0 +1,14 @@
+Content-Type: text/plain; Charset=UTF-8
```

Comment:
One single match in this diff -> I bet this EML is not needed and can be removed safely.